spec/models/test_track/notify_assignment_job_spec.rb

```diff
@@ -17,6 +17,25 @@
       assignment: assignment
     }
   end
+  let(:split_registry_url) { "http://testtrack.dev/api/v2/split_registry" }
+
+  around do |example|
+    stub_request(:get, split_registry_url)
+      .with(basic_auth: %w(dummy fakepassword))
+      .to_return(status: 200, body: {
+        splits: {
+          time: {
+            weights: {
+              back_in_time: 100,
+              power_of_love: 0
+            },
+            feature_gate: false
+          }
+        },
+        experience_sampling_weight: 1
+      }.to_json)
+    example.run
+  end
```

Comment:
can we change this to stub `TestTrack::Remote::SplitRegistry.experience_sampling_weight`? then I think we can avoid stubbing `subject` on line 76 and 118.

Reply:
k, sounds good!

Reply:
really can just stub splitregistry directly on those two lines and leave it "live" here, which works nice. did that.